packages/fxa-admin-server/scripts/email-bounce.ts

```diff
@@ -18,7 +18,7 @@ const argv = yargs.options({
   count: { type: 'number', default: 1 },
   email: { type: 'string' },
   withDiagnosticCode: { type: 'boolean' },
-}).argv;
+}).argv as any;
```

Comment:
Can avoid the cast to any by replacing `.argv` with `.parseSync()` since we don't use async commands.